src/screen/GameScreen.java

```diff
@@ -492,4 +493,14 @@ public int getLives() {
 	public void setLives(int lives) {
 		this.lives = lives;
 	}
+	
```

Comment:
It would be helpful to add descriptions for the methods.
like this
```
/**
 *
 * @param paramName 
 * @return 
 * */
public ReturnType methodName(ParamType paramName) throws ExceptionType {
    // ......
}

Reply:
Good idea